include/spatio_temporal_voxel_layer/measurement_buffer.hpp

```diff
@@ -60,14 +62,13 @@
 #include <geometry_msgs/PoseStamped.h>
 // Mutex
 #include <boost/thread.hpp>
-#include <geometry_msgs/TransformStamped.h>
```

Comment:
Removed the `TransformStamped` header?

Reply:
Putting it back (i don't know how but it was compiling without)